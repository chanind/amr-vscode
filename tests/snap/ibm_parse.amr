    # ::tok It 's reasonable for someone to call themselves an American if they 're from an American country .
    (r / reasonable-02~2
        :ARG1 (c / call-01~6
            :ARG0 (s / someone~4)
            :ARG1 s
            :ARG2 (p / person~7
                :mod (c2 / country~9
                    :name (n / name~9
                        :op1 "America"~10))))
        :condition (b / be-from-91~15
            :ARG1 s
            :ARG2 c2))