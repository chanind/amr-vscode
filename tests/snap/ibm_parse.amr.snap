>    # ::tok It 's reasonable for someone to call themselves an American if they 're from an American country .
#^^^^ source.amr
#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.amr comment.line.number-sign.amr
>    (r / reasonable-02~2
#^^^^ source.amr
#    ^ source.amr meta.structure.triple.amr punctuation.definition.triple.begin.amr
#     ^ source.amr meta.structure.triple.amr variable.other.symbol.amr
#      ^ source.amr meta.structure.triple.amr
#       ^ source.amr meta.structure.triple.amr punctuation.separator.triple.amr
#        ^ source.amr meta.structure.triple.amr
#         ^^^^^^^^^^^^^ source.amr meta.structure.triple.amr variable.other.symbol.amr
#                      ^^ source.amr meta.structure.triple.amr constant.numeric.alignment.amr
>        :ARG1 (c / call-01~6
#^^^^^^^^ source.amr meta.structure.triple.amr
#        ^^^^^ source.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#             ^ source.amr meta.structure.triple.amr meta.role.amr
#              ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.begin.amr
#               ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                 ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.separator.triple.amr
#                  ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                   ^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                          ^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr constant.numeric.alignment.amr
>            :ARG0 (s / someone~4)
#^^^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#            ^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#                 ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr
#                  ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.begin.amr
#                   ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                    ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                     ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.separator.triple.amr
#                      ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                       ^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                              ^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr constant.numeric.alignment.amr
#                                ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.end.amr
>            :ARG1 s
#^^^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#            ^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#                 ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr
#                  ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr variable.other.symbol.amr
>            :ARG2 (p / person~7
#^^^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#            ^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#                 ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr
#                  ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.begin.amr
#                   ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                    ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                     ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.separator.triple.amr
#                      ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                       ^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                             ^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr constant.numeric.alignment.amr
>                :mod (c2 / country~9
#^^^^^^^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                ^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#                    ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr
#                     ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.begin.amr
#                      ^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                        ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                         ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.separator.triple.amr
#                          ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                           ^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                                  ^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr constant.numeric.alignment.amr
>                    :name (n / name~9
#^^^^^^^^^^^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                    ^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#                         ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr
#                          ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.begin.amr
#                           ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                            ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                             ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.separator.triple.amr
#                              ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                               ^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                                   ^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr constant.numeric.alignment.amr
>                        :op1 "America"~10))))
#^^^^^^^^^^^^^^^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                        ^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#                            ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr
#                             ^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr string.quoted.double.amr
#                                      ^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr constant.numeric.alignment.amr
#                                         ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.end.amr
#                                          ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.end.amr
#                                           ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.end.amr
#                                            ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.end.amr
>        :condition (b / be-from-91~15
#^^^^^^^^ source.amr meta.structure.triple.amr
#        ^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#                  ^ source.amr meta.structure.triple.amr meta.role.amr
#                   ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.begin.amr
#                    ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                     ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                      ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.separator.triple.amr
#                       ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#                        ^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr variable.other.symbol.amr
#                                  ^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr constant.numeric.alignment.amr
>            :ARG1 s
#^^^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#            ^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#                 ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr
#                  ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr variable.other.symbol.amr
>            :ARG2 c2))
#^^^^^^^^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr
#            ^^^^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr keyword.operator.role.amr
#                 ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr
#                  ^^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr meta.role.amr variable.other.symbol.amr
#                    ^ source.amr meta.structure.triple.amr meta.role.amr meta.structure.triple.amr punctuation.definition.triple.end.amr
#                     ^ source.amr meta.structure.triple.amr punctuation.definition.triple.end.amr